### PR TITLE
Applied updates to vGPU article to make it more precise (bsc#1202224)

### PR DESCRIPTION
**Pull request by tbazant**
### PR creator: Are there any relevant issues/feature requests?

* bsc#1202224

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [x] SLE 15 SP4/openSUSE Leap 15.4
  - [x] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
  - [ ] SLE 15 SP0
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

File: xml/art-nvidia-vgpu.xml [hunk 1]
```diff
@@ -466,18 +466,18 @@ lrwxrwxrwx 1 root root           0 Sep 21 11:58 virtfn9 -> ../0000:b1:01.5
 <screen>
 &prompt.user;cd /sys/bus/pci/devices/0000:b1:00.0/virtfn1/mdev_supported_types
 &prompt.user;for i in *; do echo "" $(cat $i/name) available: $(cat $i/avail*); done
```
Comment (Julie-CAO) [inline xml/art-nvidia-vgpu.xml:468]: Please add '$' in the echo statement. It is required.

`for i in *; do echo "$i" ....`

Comment (tbazant) [inline xml/art-nvidia-vgpu.xml:468]:
```suggestion
&prompt.user;for i in *; do echo "$i" $(cat $i/name) available: $(cat $i/avail*); done
```

**Comment by Julie-CAO** [comment]
One request from the bsc#1202224 is omitted:
```
Use the MIG instance. You can use the instance directly with the UUID—for example, assign it to a VM Guest or to a container/CUDA process.
```
=>
```
Use the MIG instance. You can use the instance directly with the UUID — for example, assign it to a container/CUDA process."
```
Because the MIG instance can't be assigned to a VM Guest directly.

File: xml/art-nvidia-vgpu.xml [hunk 2]
```diff
@@ -642,12 +648,11 @@ Tue Sep 21 11:19:36 2021
     <step>
      <para>
       Use the MIG instance. You can use the instance directly with the
-      UUID&mdash;for example, assign it to a &vmguest; or to a container/CUDA
-      process.
+      UUID&mdash;for example, assign it to a container/CUDA process.
```
Comment (tbazant) [inline xml/art-nvidia-vgpu.xml:651]:
```suggestion
      UUID&mdash;for example, assign it to a container or CUDA process.
```

**Comment by Julie-CAO** [comment]
LGTM. Thanks for your effort.

**Pull request merged**